conda_press/condatools.xsh

```diff
@@ -687,8 +694,27 @@ class ArtifactInfo:
                 dep.clean()
 
 
+def get_only_deps_on_pypi(list_deps):
+    """Based on a list of dependencies this function will check if those
+    dependencies are on PyPi, if it is not available it will be removed.
+
+    :param set,list list_deps: List of dependencies as a string values
+    :return set: List of pakages present on PyPi
+    """
```

Comment:
I usually prefer to use numpydoc format.  Is that OK with you?

Reply:
Sure, no problem at all for me! I will change it